webpack.config.js

```diff
@@ -1,31 +1,37 @@
 const Encore = require('@symfony/webpack-encore');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
+if (!Encore.isRuntimeEnvironmentConfigured()) {
+    Encore.configureRuntimeEnvironment(process.env.NODE_ENV || 'dev');
+}
+
 Encore
     .setOutputPath('public/build/')
     .setPublicPath('/build')
+    .addEntry('app', './assets/app.js')
+    .enableStimulusBridge('./assets/controllers.json')
+    .splitEntryChunks()
+    .enableSingleRuntimeChunk()
     .cleanupOutputBeforeBuild()
-
-    .enableVersioning(Encore.isProduction())
-    .enableSourceMaps(!Encore.isProduction())
     .enableBuildNotifications()
-
-    .addEntry('js/app', './assets/js/app.js')
-    .configureBabel(function(babelConfig) {
-        babelConfig.presets.push('es2015');
+    .enableSourceMaps(!Encore.isProduction())
+    .enableVersioning(Encore.isProduction())
+    .configureBabel((config) => {
+        config.plugins.push('@babel/plugin-proposal-class-properties');
+    })
+    .configureBabelPresetEnv((config) => {
+        config.useBuiltIns = 'usage';
+        config.corejs = 3;
```

Comment:
This diff is messy and IMHO it does not bring anything usefull to re-ordonate line.


Plus there are suspect things
*  ```diff
    -     .addEntry('js/app', './assets/js/app.js')
    +    .addEntry('app', './assets/app.js')
    ```
    IMHO, the front is not going to work anymore

* stimulus should not be used

* `splitEntryChunks()` was not used, please remove it

* `enableSingleRuntimeChunk()` was not used, please remove it

* I'm not sure about babel conf, let's do that in another PR if needed

---

This PR is about updating SF (PHP), not the JS stack. It may be legit to update webpack encore, since it's manager by symfony. but other things must stay identical to `master`